Mathlib/Combinatorics/SimpleGraph/Clique.lean

```diff
@@ -785,6 +788,9 @@ abbrev IsIndepSet (s : Set α) : Prop :=
 theorem isIndepSet_iff : G.IsIndepSet s ↔ s.Pairwise (fun v w ↦ ¬G.Adj v w) :=
   Iff.rfl
 
+theorem isIndepSet_iff_isAntichain : G.IsIndepSet s ↔ IsAntichain G.Adj s :=
+  Iff.rfl
```

Comment:
nit:
```suggestion
  .rfl
```

Reply:
Recall that this notation wasn't available in Lean 3

Reply:
Fair point, I've updated the one above and included it in this change.